P202-Training.Domain/UsersService.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using AutoMapper;
+using P202.Training.Data;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using P202.Training.Domain.Models;
+
+namespace P202.Training.Domain
+{
+    public class UsersService : IUsersService
+    {
+        public void CreateUser(Users users)
+        {
+            if (users == null) return;
+            Mapper.Initialize(cfg => cfg.CreateMap<Users, User>());
```

Comment:
It's ok if you want to use Automapper but in that case, mapping should be configured only once. Perhaps the global.asax it's a good place to do that. And the mapper should be injected as a dependency to each of the classes that make use of it